client/types.ts

```diff
@@ -2,6 +2,7 @@
 export type URL = string;
 
 // User stuff
+export type UserId = number;
```

Comment:
this is fine for now, but later on we might discover that some user ids might be a string. I don't recommend changing anything - just wanted to point out that there are rare cases I think where this can be the case